### PR TITLE
[PyFlow]: Integrate random seed for PyFlow

### PR DESCRIPTION
**Pull request by aneels3**
Signed-off-by: aneels3 <b150023ec@nitsikkim.ac.in>

### CODE REVIEW

**Comment by mballance** [comment]
While I have no say in how well this fits with the riscv-dv project, I'd like to express my support for adding this functionality. I'm the primary developer for the pyvsc library, one of the supporting technologies used by the Python version of riscv-dv. Having seed-controlled repeatable performance results will enable me to include riscv-dv in the performance suite for pyvsc, which will benefit both projects.

File: pygen/pygen_src/test/riscv_instr_base_test.py [hunk 1]
```diff
@@ -49,6 +63,7 @@ def run_phase(self, num):
         logging.info("All directed instruction is applied")
         self.asm.gen_program()
         self.asm.gen_test_file(test_name)
+        logging.info("-----Using SEED: {}-----".format(rand_seed))
```
Comment (hodjat91) [inline pygen/pygen_src/test/riscv_instr_base_test.py:66]: This logging doesn't look good. Can you please improve it?

File: pygen/pygen_src/test/riscv_instr_base_test.py [hunk 2]
```diff
@@ -36,6 +38,18 @@ def run(self):
             pool.map(self.run_phase, list(range(cfg.num_of_tests)))
 
     def run_phase(self, num):
+        if num == 0:
+            '''Get the user specified seed value when :
```
Comment (hodjat91) [inline pygen/pygen_src/test/riscv_instr_base_test.py:42]: nit: remove the space between "when" and ":".

File: pygen/pygen_src/test/riscv_instr_base_test.py [hunk 3]
```diff
@@ -36,6 +38,18 @@ def run(self):
...
+                  1) User provides --seed through CLI
```
Comment (hodjat91) [inline pygen/pygen_src/test/riscv_instr_base_test.py:43]: nit: revisit this. "1)" isn't needed.

**Review by hodjat91 — APPROVED** [review]
Please address these few comments. Other than that, LGTM.

**Pull request merged**